lib/logstash/plugin_mixins/http_client.rb

```diff
@@ -89,6 +100,12 @@ def client_config
         c[:ssl].merge!(truststore_password: @truststore_password)
       end
     end
+    if @client_cert && @client_key
+      c[:ssl][:client_cert] = @client_cert
+      c[:ssl][:client_key] = @client_key
+    elsif !!@client_cert ^ !!@client_key
```

Comment:
Shouldn't need to use `!!` since nil is falsey, `@client_cert ^ @client_key` should be sufficient.